src/components/primitives/Accordion/Accordion.tsx

```diff
@@ -0,0 +1,38 @@
+import {
+  Accordion as ArkAccordion,
+  AccordionItem as ArkAccordionItem,
+  AccordionTrigger as ArkAccordionTrigger,
+  AccordionContent as ArkAccordionContent,
+} from "@ark-ui/react";
+
+import { panda } from "generated/panda/jsx";
+
+import type {
+  AccordionProps as ArkAccordionProps,
+  AccordionItemProps as ArkAccordionItemProps,
+  AccordionTriggerProps as ArkAccordionTriggerProps,
+  AccordionContentProps as ArkAccordionContentProps,
+} from "@ark-ui/react";
+import type { PandaComponent } from "generated/panda/types/jsx";
+
+/**
+ * Core UI accordion primitives.
+ */
+
+// [ERROR]:The inferred type of 'Accordion' cannot be named without a reference to '.store/@zag-js-accordion-npm-0.15.0-e962da3be1/node_modules/@zag-js/accordion'. This is likely not portable. A type annotation is necessary.ts(2742)
+// [FIX]: PandaComponent<typeof "Props">
```

Comment:
These comments be removed

Reply:
let me see the heat!!!!! Edits are fair game now? I love it

Reply:
:fire: :fire: